tachyon/c/math/polynomials/univariate/bn254_univariate_evaluation_domain.h

```diff
@@ -92,6 +92,19 @@ tachyon_bn254_univariate_evaluation_domain_fft(
     const tachyon_bn254_univariate_evaluation_domain* domain,
     const tachyon_bn254_univariate_dense_polynomial* poly);
 
+/**
+ * @brief Performs the in-place Fast Fourier Transform (FFT) on a given
+ * polynomial within the domain.
+ *
+ * @param domain Pointer to the evaluation domain.
+ * @param poly Pointer to the polynomial to transform.
```

Comment:
I think it would be nice to note that this poly is consumed or indicate something to make the comments different from the not `_inplace` function (+ also for the evals version)